pkg/util/log/logcrash/crash_reporting_test.go

```diff
@@ -83,7 +83,7 @@ Error types: (1) *runtime.TypeAssertionError`,
 			expErr: `some visible detail: interface conversion: interface {} is nil, not int
 (1) attached stack trace
   -- stack trace:
-  | github.com/cockroachdb/cockroach/pkg/util/log/logcrash.init.func2
+  | github.com/cockroachdb/cockroach/pkg/util/log/logcrash.init.func3
```

Comment:
I believe these names are generated on compilation, I'm assuming that somehow by importing another package it changed these names